rickshaw.js

```diff
@@ -1938,7 +1934,7 @@ Rickshaw.Graph.Axis.Y = Rickshaw.Class.create({
 
     if (this.orientation == 'left') {
       var berth = this.height * this.berthRate
```

Comment:
is the `berth` declaration used anywhere else in `_drawAxis`'s scope? if not, remove. same goes for `this.berthRate`

Reply:
edit - I should clarify: remove `this.berthRate` if it is not used anywhere else in the file, not just _drawAxis.

Reply:
`this.berthRate` is still used to calculate axis height, but have removed unused `berth`s in `_drawAxis` and `setSize`